pkg/graveler/graveler.go

```diff
@@ -1591,6 +1606,9 @@ func (g *Graveler) LoadCommits(ctx context.Context, repositoryID RepositoryID, m
 		for i, p := range commit.GetParents() {
 			parents[i] = CommitID(p)
 		}
+		if commit.GetGeneration() == 0 {
```

Comment:
Use a named constant?

Reply:
Constant for what? 0 is just the default value for int in this context